webscrapbook/scrapbook/common/util.py

```diff
@@ -0,0 +1,121 @@
+import os, json, glob, re
+from math import ceil
+from collections import OrderedDict
+from itertools import islice
+
+# string functions
+###############################################################################
+
+def remove_prefix(text, prefix):
+    if text.startswith(prefix):
+        return text[len(prefix):]
+    return text
+
+def remove_suffix(text, suffix):
+    if text.endswith(suffix):
+        return text[:-1 * len(suffix)]
+    return text
+
+def remove_lines(file, count):
+    for _ in range(count):
+        file.readline()
+
+def parse_json(filepath, preprocessing):
+    data = {}
+    with open(filepath) as file:
+        json_string = preprocessing(file)
+        try:
+            data = json.loads(json_string, object_pairs_hook=OrderedDict)
+        except:
+            # TODO: better handling
+            raise Exception("Failed to parse json file " + filepath)
+    return data
+
+def get_number_suffix(text):
+    ''' return the number at the end of a string if no number found return empty string '''
+    return re.findall('([0-9]*$)', text)[0]
+
+###############################################################################
+
+
+# filesystem functions
+###############################################################################
+
+def add_directory_to_filename(directory, filename):
+    return os.path.join(directory, filename)
```

Comment:
Such helper function is kind off too basic. I'm not sure if we really need it.

Reply:
True, I will get rid of it.

Reply:
Fixed!